src/test/scala/uk/gov/hmrc/agentmtdidentifiers/model/PptRefSpec.scala

```diff
@@ -16,23 +16,21 @@
 
 package uk.gov.hmrc.agentmtdidentifiers.model
 
-import org.scalacheck.Gen
 import org.scalatest.{FlatSpec, Matchers}
 import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
 
 class PptRefSpec extends FlatSpec with Matchers with ScalaCheckPropertyChecks {
 
-  val permittedChars = Gen.oneOf("ABCDEFGHIJKLMNOQPRSTUVWXYZ0123456789")
-  val validPptId = Gen.listOfN(15, permittedChars).map(_.toArray).map(new String(_))
-
   it should "be true for a valid PPT" in {
-    forAll(validPptId) { pptId =>
-      PptRef.isValid(pptId) shouldBe true
-    }
+      PptRef.isValid("XAPPT000000000") shouldBe true
```

Comment:
Where is the confirmation it is 14 instead of 15 digits? The latest slack message from Paul is confusing as they state the length is still 15 there - but the regex given matches only 14 digits unless I'm reading this wrong

Reply:
now 15, thx